samples/matter/common/src/bridge/bridge_manager.cpp

```diff
@@ -307,30 +316,39 @@ CHIP_ERROR BridgeManager::AddDevices(MatterBridgedDevice *devices[], BridgedDevi
 				     uint8_t deviceListSize, chip::Optional<uint8_t> devicesPairIndexes[],
 				     uint16_t endpointIds[])
 {
-	VerifyOrReturnError(devices && dataProvider, CHIP_ERROR_INTERNAL);
+	CHIP_ERROR err{ CHIP_NO_ERROR };
+	CHIP_ERROR addingStatus{ CHIP_NO_ERROR };
+
+	VerifyOrReturnError(devices || dataProvider, CHIP_ERROR_INTERNAL);
```

Comment:
CHIP_ERROR_INTERNAL -> CHIP_ERROR_INVALID_ARGUMENT? And the same to the below ones.